.github/workflows/test.yml

```diff
@@ -1,7 +1,7 @@
 name: Test
```

Comment:
I believe we have test runs with Travis already. Do we need to have another test run in GitHub workflows? Is not this an extra?

Reply:
True, I don't remember why is this stage duplicated